src/main/java/com/iexec/standalone/chain/IexecHubService.java

```diff
@@ -67,6 +75,20 @@ public IexecHubService(CredentialsService credentialsService,
         }
     }
 
+    @Autowired
+    public IexecHubService(CredentialsService credentialsService,
```

Comment:
This constructor should not be necessary, the Spring bean would already be created with the previous constructor